diff --git a/node_modules/prettier-plugin-toml/lib/index.js b/node_modules/prettier-plugin-toml/lib/index.js
index 36d733b..a1b71d2 100644
--- a/node_modules/prettier-plugin-toml/lib/index.js
+++ b/node_modules/prettier-plugin-toml/lib/index.js
@@ -5,6 +5,10 @@ import { languages } from './languages.js';
 import { prettierOptionsDefinitions } from './options.js';
 import { loadWasm } from './wasm.js';
 const PLUGIN_NAME = 'toml';
+
+// 32 MiB in bytes (characters)
+const MAX_TOML_FILE_SIZE = 32 * 1024 * 1024; // 33 554 432
+
 let tombiPromise;
 class TombiFormatError extends SyntaxError {
     constructor(diagnostic) {
@@ -60,8 +64,8 @@ const TomlPlugin = {
         [PLUGIN_NAME]: {
             parse: (code, options) => format(code, options),
             astFormat: PLUGIN_NAME,
-            locStart: () => -1,
-            locEnd: () => -1,
+            locStart: () => 0,
+            locEnd: () => MAX_TOML_FILE_SIZE,
         },
     },
     printers: {
