diff --git a/node_modules/prettier/index.mjs b/node_modules/prettier/index.mjs
index 0c03045..293e2b6 100644
--- a/node_modules/prettier/index.mjs
+++ b/node_modules/prettier/index.mjs
@@ -20083,7 +20083,11 @@ async function textToDoc(text, partialNextOptions, parentOptions, printAstToDoc2
       ...parentOptions,
       ...partialNextOptions,
       parentParser: parentOptions.parser,
-      originalText: text
+      originalText: text,
+      // Improve this if we calculate the relative index
+      cursorOffset: void 0,
+      rangeStart: void 0,
+      rangeEnd: void 0
     },
     { passThrough: true }
   );
@@ -20129,11 +20133,13 @@ async function printAstToDoc(ast, options8) {
     embeds
   );
   ensureAllCommentsPrinted(options8);
-  if (options8.nodeAfterCursor && !options8.nodeBeforeCursor) {
-    return [cursor, doc2];
-  }
-  if (options8.nodeBeforeCursor && !options8.nodeAfterCursor) {
-    return [doc2, cursor];
+  if (options8.cursorOffset >= 0) {
+    if (options8.nodeAfterCursor && !options8.nodeBeforeCursor) {
+      return [cursor, doc2];
+    }
+    if (options8.nodeBeforeCursor && !options8.nodeAfterCursor) {
+      return [doc2, cursor];
+    }
   }
   return doc2;
   function mainPrint(selector, args) {
